content/open-source/guides/learn-godot/beginner/index.md

```diff
@@ -0,0 +1,106 @@
+
+I cannot tell whether you will be able to create your dream project or not, depending on how crazy it is. Still, you can undoubtedly make games that people will enjoy, even alone. It will just take some time and dedication.
+
+> Great things are not done by impulse, but by a series of small things brought together.
+> 
+> Vincent Van Gogh
+
+
+## Getting started with game development
+
+Many enthusiasts starting on a journey to learn a new topic or skill **quickly become swamped by the sheer amount of information out there**. Nowadays, the problem isn't access to information as it once was, but rather **too much scattered information**. There's also the problem that most of it **isn't exactly respectable quality**, because anyone can put anything on the internet without any filters and quality control.
+
+That said, **we, here at [GDQuest](https://www.gdquest.com/), have done the work for you** by creating this **curated list of free online resources** that get you from **no-programming experience to your first full-blown game projects!**
+
+_Buckle your seatbelts, it going to be an awesome ride!_
+
+### Introduction: the first step in a thousand mile journey
+
+This section is for the total beginner that has little to no idea about the game industry nor the game creation process. It introduces high-level concepts that give some insights into what is to be expected.
+
+1. [GDQuest](https://www.gdquest.com/)'s very own starter guide - [Become a Game Developer: How to Learn to Code]({{< ref "tutorial/game-design/godot/become-a-game-creator/first-steps-programming/index.md" >}})
+1. [Sykoo](https://www.youtube.com/user/SykooTV/)'s [How To Get into Game Development! (Teachers, School, Self-Taught and MORE!)](https://youtu.be/EL_Le8B_iCo)
+1. [GDQuest](https://www.gdquest.com/)'s [Game Engines: what are they and how to choose the right one?](https://youtu.be/2tZK75R2K2c)
+
+### Getting started with [Godot](https://godotengine.org/) & your first actual game(s)
+
+Instead of the booooring step-by-step traditional approach of going over the theory and explaining the interface of the engine, then going through the features etc. we're going to **focus on a hands-on approach!** Best way of learning something new is through a balanced practice-theory approach. That's why we have also included a few resources that do go through the theory, but **our recommendation is to start off with the heavy hands-on resources first** and if you get stuck or if you want to understand some game engine editor & programming concepts, then go through the theory and abstract explanations.
+
+1. [GDQuest](https://www.gdquest.com/)'s [Intro to GDScript for Programming Beginners](https://youtu.be/UcdwP1Q2UlU) - **fundamentals of programming using GDScript**, the [Godot](https://godotengine.org/) scripting language. This is _mostly optional_ for people that have basic programming knowledge
```

Comment:
I'd say it's still important for people with basic programming knowledge, especially as it looks like Python but it's not Python. I had feedback from a tutor who tried to teach Python then have the students move to GDScript and they had a lot of issues making the leap.

Reply:
Alright, I can see how that works.